packages/twenty-server/.env.example

```diff
@@ -1,13 +1,22 @@
 # Use this for local setup
-PG_DATABASE_URL=postgres://twenty:twenty@localhost:5432/default
+# PG_DATABASE_URL=postgres://twenty:twenty@localhost:5432/default
 # Use this for docker setup
-# PG_DATABASE_URL=postgres://twenty:twenty@postgres:5432/default
+PG_DATABASE_URL=postgres://twenty:twenty@postgres:5432/default
```

Comment:
@AutoComput we optimize for yarn install for local contributors. I think we should not mix docker-compose .env (infra level) with this twenty-server .env (app-level)

We can either maintain two .env or make a small script to copy it add TAG=latest or any variable needed by docker-compose and update docker-compose self host documentation